### PR TITLE
Some StaffWebUI updates

### PR DESCRIPTION
**Pull request by vnbaaij**
Make it all a bit more Fluent
- Update Fluent UI packages
- Use FluentDataGrid for Tickets
- Update layout and styling
- Update vulnerable packages (STJ)

### CODE REVIEW

**Comment by SteveSandersonMS** [comment]
> Use FluentDataGrid for Tickets

Just to clarify on this, I did implement it with FluentDataGrid originally, but I found it to be laggy. Switching to QuickGrid resolved this. I'd be happy to go back to FluentDataGrid if the performance is the same and we can quantify this.

> Update layout and styling

Would you be able to post before/after screenshots of each change? It's very hard to understand the code diff otherwise.

File: src/StaffWebUI/wwwroot/app.css [hunk 1]
```diff
@@ -3,10 +3,6 @@ html.dark-mode {
     color-scheme: dark;
 }
 
-html:has(.ensure-page-scrollbar) {
-    overflow-y: scroll;
-}
-
```
Comment (SteveSandersonMS) [inline src/StaffWebUI/wwwroot/app.css:9]: Why remove this?

Comment (vnbaaij) [reply src/StaffWebUI/wwwroot/app.css:9]: I've moved the scroll bar from being on the whole right side/page to just be on the tickets grid. So there is now basically one big sticky header with the title and tabs/filter.

Comment (SteveSandersonMS) [reply src/StaffWebUI/wwwroot/app.css:9]: Can you still scroll with keyboard just using cursor keys when you first arrive at the page?

Comment (vnbaaij) [reply src/StaffWebUI/wwwroot/app.css:9]: No, you'd need to tab into the grid first. Works, but is more cumbersome than how it was originally. 

I can change it back to original behavior, but then I can't get the header on the `FluentDataGrid` to stick. That is annoying but far less intrusive then not being able to use cursor keys immediately. You probably do not want to start a sort when you're half way down the list of tickets.

Comment (SteveSandersonMS) [reply src/StaffWebUI/wwwroot/app.css:9]: Do you mean the column headers would become invisible once you start to scroll down?

Comment (vnbaaij) [reply src/StaffWebUI/wwwroot/app.css:9]: Yes:
image

Comment (SteveSandersonMS) [reply src/StaffWebUI/wwwroot/app.css:9]: Sorry, I don't think we can do that. Losing track of what the columns mean harms the usability too much.

File: src/StaffWebUI/Components/Pages/Tickets/TicketsFilter.razor.css [hunk 2]
```diff
@@ -3,7 +3,7 @@
     position: sticky;
     top: var(--header-height);
     background: var(--neutral-fill-stealth-active);
-    z-index: 2;
+    z-index: 4;
```
Comment (SteveSandersonMS) [inline src/StaffWebUI/Components/Pages/Tickets/TicketsFilter.razor.css:6]: Is this needed due to a change in the FluentUI packages? Fine if so, just want to understand.

Comment (vnbaaij) [reply src/StaffWebUI/Components/Pages/Tickets/TicketsFilter.razor.css]: This was for the sticky header to appear below the filter when using the `FluentDataGrid`

**Pull request merged**